ignite/handlers/checkpoint.py

```diff
@@ -7,10 +7,11 @@
 from abc import ABCMeta, abstractmethod
 from collections import OrderedDict
 from pathlib import Path
-from typing import Any, Callable, Dict, IO, List, Mapping, NamedTuple, Optional, Tuple, Union
+from typing import Any, Callable, Dict, List, Mapping, NamedTuple, Optional, Tuple, Union
 
 import torch
 import torch.nn as nn
+from torch.distributed.optim import ZeroRedundancyOptimizer
```

Comment:
Ignite is supposed to work with pytorch 1.3.1 where `ZeroRedundancyOptimizer` is absent. We have to deal with this import differently.